internal/config/helpers.go

```diff
@@ -26,6 +26,24 @@ func Environment() string {
 	return "dev"
 }
 
+// EnvironmentPrefix wraps an identifier (e.g. id, unique id) in the following way.
+// For production environment, it returns "prefix-identifier". For any other environment,
+// it returns "prefix-identifier-env". Examples:
+//
+// * reservation-14572
+// * reservation-9531-stage
+// * reservation-13-ephemeral
+// * reservation-1-dev
+func EnvironmentPrefix(prefix, identifier string) string {
+	env := Environment()
+
+	if strings.HasPrefix(env, "prod") || env == "" {
+		return prefix + "-" + identifier
+	}
+
+	return prefix + "-" + identifier + "-" + env
```

Comment:
There could be a way where the prefix and the identifier would be empty strings, and therefore, the prefix would be something like "--" + env, but I guess we do not mind?

Reply:
Well trash in - trash out :-) Yeah minor issue.